ios/FluentUI/ActivityIndicator/ActivityIndicatorTokens.swift

```diff
@@ -16,40 +16,21 @@ import UIKit
 }
 
 /// Design token set for the `ActivityIndicator` control.
-public class ActivityIndicatorTokens: ControlTokens {
-    /// Creates an instance of `ActivityIndicatorTokens` with optional token value overrides.
-    /// - Parameters:
-    ///   - size: MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
-    ///   - defaultColor: The default color of the Activity Indicator.
-    ///   - side: The value for the side of the square frame of an Activity Indicator.
-    ///   - thickness: The value for the thickness of the ActivityIndicator ring.
-    public init(size: MSFActivityIndicatorSize,
-                defaultColor: DynamicColor? = nil,
-                side: CGFloat? = nil,
-                thickness: CGFloat? = nil) {
-
-        self.size = size
-        super.init()
-
-        // Optional overrides
-        if let defaultColor = defaultColor {
-            self.defaultColor = defaultColor
-        }
-        if let side = side {
-            self.side = side
-        }
-        if let thickness = thickness {
-            self.thickness = thickness
-        }
+open class ActivityIndicatorTokens: ControlTokens {
+    /// MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
+    public var size: MSFActivityIndicatorSize {
+        return state.size
```

Comment:
This points to ta property defined in the state class and it's of a type that is defined in the Tokens class.

I think it makes a better cohesion story to keep this all in the Tokens space and have the state use it to prevent the cross-dependency between them.

Reply:
I think I can get behind that argument, but I don't think we can just have `state` refer to the value in its `tokens`, as that would mean that `style` and `size` would reset any time a client provided custom tokens, either directly or through the current theme. So we'll need to maintain two sets of values and do a `didSet` dance in the `state` class to keep them in sync.